db/migrate/20200906033932_squasher_clean.rb

```diff
@@ -0,0 +1,13 @@
+class SquasherClean < ActiveRecord::Migration[6.0]
+  class SchemaMigration < ActiveRecord::Base
+  end
+
+  def up
+    migrations = Dir.glob(File.join(File.dirname(__FILE__), '*.rb'))
+    versions = migrations.map { |file| File.basename(file)[/\A\d+/] }
+    SchemaMigration.where("version NOT IN (?)", versions).delete_all
```

Comment:
This will remove migrations that came from plugins, causing them to be executed again. 
TBH, I'm not sure we have to clean up the squashed migrations from the `schema_migrations` table - rails is perfectly fine with having migrations listed in the table that have been removed - see https://edgeguides.rubyonrails.org/active_record_migrations.html#old-migrations.

Reply:
Yes, I also had some issues with that. It's generated from https://github.com/jalkoby/squasher/blob/master/lib/squasher/templates/squasher_clean.rb.erb. I also thought about an explicit list of ones that we know have been removed.

Reply:
Thinking more about it, it may be good to keep a record of that. If something ever shows up, you at least know which migration path it took.